scripts/telegraf.service

```diff
@@ -14,6 +14,8 @@ Restart=on-failure
 RestartForceExitStatus=SIGPIPE
 KillMode=control-group
 LimitMEMLOCK=8M:8M
+PrivateMounts=true
```

Comment:
From the [man-page](https://www.freedesktop.org/software/systemd/man/systemd.unit.html):
> Unit files may contain additional options on top of those listed here. If systemd encounters an unknown option, it will write a warning log message but continue loading the unit. 

Should I comment the option the same way as `LoadCredentialEncrypted`?

Reply:
Yeah - can we comment out the option and make this opt-in please?

Reply:
Well, `PrivateMounts` is an important security feature. I noticed that grafana's systemd unit packaging has directives that for newer `systemd` and when using the unit on an old `systemd`, `systemd` complains in the logs, but otherwise ignores it. Is this an option (I've not tested the behavior with `PrivateMounts`)?

Reply:
I have an absolute fear of regressing or changing the package behavior. As both have you have now mentioned it should just ignore them, so maybe I am being a little too aggressive with avoiding the change if that is the case. Happy to keep it then.